data-upgrade-portal-rdbms/pom.xml

```diff
@@ -19,9 +19,9 @@
     <version>6.3.x-maintenance-SNAPSHOT</version>
```

Comment:
you should change also the name of the module

Reply:
it is branch based on feauture/maintenance, i think we shouldn't change it.

Reply:
why not ?